storage/filestore/dir.go

```diff
@@ -398,8 +513,14 @@ func (dir *Dir) ListNamespaces(ctx context.Context) (ids [][]byte, err error) {
 // iterating and return the error immediately. The ctx parameter is intended specifically to allow
 // canceling iteration early.
 func (dir *Dir) WalkNamespace(ctx context.Context, namespace []byte, walkFunc func(storage.BlobInfo) error) (err error) {
+	defer mon.Task()(&ctx)(&err)
+	return dir.walkNamespaceInPath(ctx, namespace, dir.blobsdir(), walkFunc)
```

Comment:
Do we need this empty function? Why not put the implementation in here? It doesn’t hurt anything and shouldn’t prevent the merge but we do a lot of this I’ve noticed and it really bloats and causes a lot of excessive nesting in our monkit traces because we have a lot of functions that have no code in them except a monkit trace and it calls another function that also monkit traces.

Reply:
There is code, this function is the exported version that always walks over the default `blobsdir`, whereas the more generic version that I created can walk a namespace in any sub-dir, which is how I'm using it in `RestoreTrash`.